### DISCUSSION TITLE
en/one_click_deployment/os/

### DISCUSSION THREAD

**Discussion by giscus** [cargo-youth/ToolDiy#56] [Announcements]
# en/one_click_deployment/os/

https://tooldiy.ry.rs/en/one_click_deployment/os/

**Comment by Mc-GrowlR** [comment]
对于 `Arch` 用户，可以一条命令安装 `qemu` （版本为 `7.2.0`）：
``` shell
yay -S qemu-full
```
**备注**： [QEMU = 7.0.0 修改了拆包方式 « Arch Linux](https://www.archlinuxcn.org/qemu-700-changes-split-package-setup/)。原本下载的 `Virtio` 的驱动示例不能显示图形界面，在我将包 `qemu-base` 换成了 `qemu-full` 之后便解决了。
包管理器会自动检测依赖，很方便。
还有 aur 源也提供了 `RISC-V ToolChain` 工具链的包。

**Reply by storage-db** [reply]
好的感谢你的反馈

**Reply by Mc-GrowlR** [reply]
补充：[在Arch 上安装Risc-v工具链](https://blog.csdn.net/GrowlR/article/details/129179281)
